src/lib.rs

```diff
@@ -177,6 +177,8 @@
 //!     // Get a serialized version of the input data as a `Bson`.
 //!     let redacted_bson = bson::to_bson(&person).unwrap();
 //! }
+//!
+//! typed_example();
```

Comment:
@patrickfreed I fixed the doc test. I think this line is needed, otherwise the code won't be exercised. Let me know if you see a problem. Thanks!

Reply:
Oh good point. In fact, I think this example should probably just not be included in a `fn` at all, since I don't think it really adds anything. So could you just remove the `typed_example` function wrapping here and in the README?

Reply:
Are these examples getting run, or just compiled? I thought they were just compiled by default

Reply:
They're wrapped in an implicit `fn main() {}`, compiled, and then run. If they don't panic, they pass.

ref: https://doc.rust-lang.org/rustdoc/documentation-tests.html#passing-or-failing-a-doctest

Reply:
That's a good idea. I've removed the wrapping function in 0885ae0.

Reply:
Hmm, why do the driver doc tests work when no database is running then?

Reply:
Oh wait, for the same reason as here; we don't call the functions that we define in the examples

Reply:
@saghm I think fixing the other code in doc tests is beyond the scope of this PR. Does this PR look good for merging?